data/json/npcs/TALK_TEST.json

```diff
@@ -682,84 +682,84 @@
     "responses": [
       { "text": "This is a basic test response.", "topic": "TALK_DONE" },
       {
-        "text": "This is a u_adjust_var test response that increments by 1.",
+        "text": "This is a math test response that increments by 1.",
```

Comment:
Better remove this object (`TALK_TEST_ADJUST_VARS`) and its associated test (`npc_talk_adjust_vars`).